lua/wikis/commons/TournamentsListing/CardList.lua

```diff
@@ -528,6 +528,7 @@ end
 ---@return Html
 function BaseTournamentsListing:displayTier(tournamentData)
 	local tier, tierType, options = Tier.parseFromQueryData(tournamentData)
+	---@cast tier -nil
```

Comment:
This doesn't look like good practice, update annos on `parseFromQueryData` instead

Reply:
technicallly it can be nil on smash like wikis iirc
should we just ignore them for this?